From 788454645497907864df818dcdd7e2fc51e3c7cf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonathan=20Grotel=C3=BCschen?= <tippfehlr@tippfehlr.eu>
Date: Tue, 26 May 2026 16:52:42 +0200
Subject: [PATCH] fix: set RESOURCES_PATH relative to app.getAppPath()

When running the app with system electron, process.resourcesPath points
to the resources folder of the system electron, not the one from
feishin.
---
 src/main/index.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/index.ts b/src/main/index.ts
index 262f0046..65921dc3 100644
--- a/src/main/index.ts
+++ b/src/main/index.ts
@@ -335,7 +335,7 @@ if (isDevelopment) {
 }
 
 const RESOURCES_PATH = app.isPackaged
-    ? path.join(process.resourcesPath, 'assets')
+    ? path.join(path.dirname(app.getAppPath()), 'assets')
     : path.join(__dirname, '../../assets');
 
 const getAssetPath = (...paths: string[]): string => {
-- 
2.54.0

